crates/forge_analyzer/src/checkers.rs

```diff
@@ -1103,7 +1103,15 @@ impl<'cx> Dataflow<'cx> for PermissionDataflow {
                             first_arg_vec.iter().for_each(|first_arg| {
                                 let first_arg = first_arg.replace(&['\"'][..], "");
                                 second_arg_vec.iter().for_each(|second_arg| {
-                                    if intrinsic_func_type
+                                    if intrinsic_func_type == IntrinsicName::RequestJiraSoftware {
+                                        let permissions = check_url_for_permissions(
+                                            interp.jira_software_permission_resolver,
+                                            interp.jira_software_regex_map,
```

Comment:
I think a match statement for `intrinsic_func_type` would be more idiomatic here. I realize this was originally my code, sorry 😓.

Also, maybe we have the second argument be set to get if it does not exists. That eliminates the need for the first if else block.

Reply:
These may be out of scope for this PR, and I can make the changes since this was originally my bad code.

Reply:
@jwong101 @jwong33